core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/colsOfKind.kt

```diff
@@ -199,9 +199,9 @@ public interface ColsOfKindColumnsSelectionDsl {
  * @param filter The filter function to apply on each column. Must accept a ColumnWithPath object and return a Boolean.
  * @return A [TransformableColumnSet] containing the columns of given kinds that satisfy the filter.
  */
-internal fun ColumnsResolver<*>.columnsOfKindInternal(
+internal inline fun ColumnsResolver<*>.columnsOfKindInternal(
```

Comment:
currently there's no benefit for having these CS DSL internal functions inlined, because they are only called from non-inlined functions. However, I think they can stay inlined, because it reminds us we can move the CS DSL to a fully inlined context-parameter based API in the future :)